Tests/NIOTests/DatagramChannelTests.swift

```diff
@@ -106,15 +106,24 @@ final class DatagramChannelTests: XCTestCase {
     private var firstChannel: Channel! = nil
     private var secondChannel: Channel! = nil
 
-    private func buildChannel(group: EventLoopGroup) throws -> Channel {
+    private func buildChannel(group: EventLoopGroup, host: String = "127.0.0.1") throws -> Channel {
         return try DatagramBootstrap(group: group)
             .channelOption(ChannelOptions.socketOption(.so_reuseaddr), value: 1)
             .channelInitializer { channel in
                 channel.pipeline.addHandler(DatagramReadRecorder<ByteBuffer>(), name: "ByteReadRecorder")
             }
-            .bind(host: "127.0.0.1", port: 0)
+            .bind(host: host, port: 0)
             .wait()
     }
+    
+    private var supportsIPv6: Bool {
+        do {
+            let ipv6Loopback = try SocketAddress.makeAddressResolvingHost("::1", port: 0)
```

Comment:
This shouldn't be `makeAddressResolvingHost`, we don't need to involve `getaddrinfo` in this. We can use `SocketAddress(ipAddress:port:)`.

Reply:
Again, I note you copied it from elsewhere in the code, but now that we've spotted it let's fix it up (I think the original use-case predates this initializer, in fact.)